wiki/content/deploy/index.md

```diff
@@ -1537,14 +1537,14 @@ $ dgraph cert --help
 # Create Dgraph Root CA, used to sign all other certificates.
 $ dgraph cert
 
-# Create node certificate (needed for Dgraph Live Loader using TLS)
-$ dgraph cert -n live
+# Create node certificate
+$ dgraph cert -n localhost
 
-# Create client certificate
+# Create client certificate for mTLS
```

Comment:
Create client certificate and key for mTLS (mutual TLS)

Reply:
This is resolved in a later commit, I am not sure why this is still unresolved.